### PR TITLE
added routes for admin, user and artists

### PR DESCRIPTION
**Pull request by Omotunde-G**

### CODE REVIEW

File: backend/db/index.example.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const { Client } = require("pg");
+require("dotenv").config();
+const client = new Client({
+  user: "user",
+  host: "localhost",
+  database: "savanna-showcase",
+  password: process.env.password,
+  port: 5432,
+});
+console.log();
```
Comment (salimkarbm) [inline backend/db/index.example.js]: remove the console and put all your database credentials inside the env file

File: backend/controller/auth.js [hunk 2]
```diff
@@ -1,20 +1,18 @@
...
     const { name, email, password } = req.body;
     const hashedPassword = await hashPassword(password, saltRounds);
   
-    const result = db.query(
-      "INSERT INTO users (name, email, password) VALUES ($1, $2, $3) RETURNING *",
-      [name, email, hashedPassword]
-    );
-    console.log(hashPassword);
+    const result = await usermodel.createUser(name, email, hashedPassword)
+    console.log(hashedPassword);
```
Comment (salimkarbm) [inline backend/controller/auth.js]: remove the console

**Comment by Omotunde-G** [comment]
@salimkarbm 
please review

File: backend/controller/adminController.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+const userModel = require('../models/userModel')
+
+async function createNewUser(req, res, next) {
+    try {
+        //  check if the person is an admin
```
Comment (salimkarbm) [inline backend/controller/adminController.js]: remove dead codes, i.e code that you are not using.

File: backend/controller/user.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+const {updateUserProfile} = require("../models/userModel")
+
+async function updateProfile(req, res) {
+    const updatedFields = req.body;
+    console.log(updatedFields);
```
Comment (salimkarbm) [inline backend/controller/user.js]: remove console log

File: backend/models/adminmodel.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
+const db = require("../db")
+
+
+async function createUser(name, email, password, user_role) {
+  try {
+    const result = await db.query(
```
Comment (salimkarbm) [inline backend/models/adminmodel.js:6]: you database queries should be in the models

**Comment by Omotunde-G** [comment]
@salimkarbm please rewiew

**Pull request closed without merging**